### PR TITLE
`DOMString` -> `string`

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #15499 

### Summary
`DOMString` is not exposed by browsers to JavaScript. It is converted to `string`.[[ref](https://developer.mozilla.org/en-US/docs/Web/API/DOMString)]

Same is the case with `USVString` and `CSSOMString`.

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/mediastream/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ _This interface inherits properties from its parent, {{domxref("EventTarget")}}.
 - {{domxref("MediaStream.active")}} {{readonlyinline}}
   - : A Boolean value that returns `true` if the `MediaStream` is active, or `false` otherwise.
 - {{domxref("MediaStream.id")}} {{readonlyInline}}
-  - : A {{domxref("DOMString")}} containing 36 characters denoting a universally unique identifier ({{Glossary("UUID")}}) for the object.
+  - : A string containing 36 characters denoting a universally unique identifier ({{Glossary("UUID")}}) for the object.
```
Comment (wbamberg) [inline files/en-us/web/api/mediastream/index.md:35]:
```suggestion
  - : A string containing a 36-character universally unique identifier ({{Glossary("UUID")}}) for the object.
```

File: files/en-us/web/api/mediastreamtrack/kind/index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ const type = track.kind
 
 ### Value
 
-The possible values are a {{domxref("DOMString")}} with on of the following values:
+The possible values are a string with on of the following values:
```
Comment (wbamberg) [inline files/en-us/web/api/mediastreamtrack/kind/index.md:29]:
```suggestion
The possible values are a string with one of the following values:
```

File: files/en-us/web/api/mediatrackconstraints/index.md [hunk 3]
```diff
@@ -55,9 +55,9 @@ The `ConstrainDouble` constraint type is used to specify a constraint for a prop
 The `ConstrainDOMString` constraint type is used to specify a constraint for a property whose value is a string. Its value may either be set to a string, an array of strings, or an object containing the following properties:
 
 - `exact`
-  - : A {{domxref("DOMString")}}, or an array of `DOMString`s, one of which must be the value of the property. If the property can't be set to one of the listed values, matching will fail.
+  - : A string, or an array of `DOMString`s, one of which must be the value of the property. If the property can't be set to one of the listed values, matching will fail.
```
Comment (wbamberg) [inline files/en-us/web/api/mediatrackconstraints/index.md:58]:
```suggestion
  - : A string or an array of strings, one of which must be the value of the property. If the property can't be set to one of the listed values, matching will fail.
```

File: files/en-us/web/api/mediatrackconstraints/index.md [hunk 4]
```diff
@@ -55,9 +55,9 @@ The `ConstrainDouble` constraint type is used to specify a constraint for a prop
...
 - `ideal`
-  - : A {{domxref("DOMString")}}, or an array of `DOMString`s, specifying ideal values for the property. If possible, one of the listed values will be used, but if it's not possible, the user agent will use the closest possible match.
+  - : A string, or an array of `DOMString`s, specifying ideal values for the property. If possible, one of the listed values will be used, but if it's not possible, the user agent will use the closest possible match.
```
Comment (wbamberg) [inline files/en-us/web/api/mediatrackconstraints/index.md:60]:
```suggestion
  - : A string or an array of strings, specifying ideal values for the property. If possible, one of the listed values will be used, but if it's not possible, the user agent will use the closest possible match.
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @OnkarRuikar !

**Comment by OnkarRuikar** [comment]
@wbamberg thanks! 
I've applied the review comments.

**Review by wbamberg — APPROVED** [review]
👍 thank you @OnkarRuikar !

**Pull request merged**